app/assets/javascripts/application.js

```diff
@@ -13,7 +13,7 @@
 //= require jquery
 //= require jquery.turbolinks
 //= require jquery_ujs
-//= require dataTables/jquery.dataTables
-//= require turbolinks
 //= require_tree .
 //= require gulp/app
+//= require dataTables/jquery.dataTables
+//= require turbolinks
```

Comment:
I think `gulp/app` should come last, since that's our code, and `require_tree .` should come right before that. It's possible things get messed up if we use something in `dataTables/jquery.dataTables` if it isn't declared beforehand, although our code isn't affected yet, since we've been hooking onto `$(document).ready(...)`.